test-app/tests/integration/components/headless-form-custom-controls-test.gts

```diff
@@ -0,0 +1,381 @@
+/* eslint-disable no-undef -- Until https://github.com/ember-cli/eslint-plugin-ember/issues/1747 is resolved... */
```

Comment:
I wonder how close we are to the 200 repos needed for github to add highlighting for this file format 🤔 

https://github.com/github/linguist/blob/master/CONTRIBUTING.md#adding-an-extension-to-a-language

Reply:
https://github.com/ember-template-imports/ember-template-imports/issues/123